frontend/packages/jobs/src/prompts/generateDocsSuggestion/generateDocsSuggestion.ts

```diff
@@ -83,6 +84,8 @@ ${DOCS_STRUCTURE_DESCRIPTION}
 
 </text>
 
+{reviewFeedbackInfo}
+
```

Comment:
Sorry, the process of reading reviewFeedbacks is put in this PR. Could you please use this one?

https://github.com/liam-hq/liam/pull/1359

Reply:
I think I could have handled it!🙏
If you could take a look at `file changed` I would appreciate it.

https://github.com/liam-hq/liam/pull/1371/files

I was able to confirm this locally.
https://github.com/liam-hq/liam/pull/1371#issuecomment-2812532003